docs/src/main/asciidoc/security-vulnerability-detection.adoc

```diff
@@ -130,9 +130,10 @@ Ensure that you review and update the suppression list regularly to ensure that
 You can optionally apply a time limit to individual suppressions by adding an expiry attribute, as outlined in the following example:
 
 `<suppress until="2022-01-01Z">...</suppress>`
+
 You can adjust the expiry date if you need to.
 
 == References
 
 * xref:security-overview.adoc[Quarkus Security overview]
-* xref:security-authentication-mechanisms.adoc#other-supported-authentication-mechanisms[Authentication mechanisms in Quarkus]
+* xref:security-authentication-mechanisms.adoc#other-supported-authentication-mechanisms[Other supported authentication mechanisms]
```

Comment:
```suggestion
* xref:security-authentication-mechanisms.adoc[Supported authentication mechanisms]
```

Reply:
And the doc title is `Authentication mechanisms in Quarkus` so I just updated the suggestion

Reply:
Fixed.